### PR TITLE
Enitity Type resource

### PR DESCRIPTION
**Pull request by masha1997**

### CODE REVIEW

File: provider/datasource_entity_type.go [hunk 1]
```diff
@@ -43,29 +38,5 @@ func entityTypeRead(ctx context.Context, d *schema.ResourceData, m interface{})
...
-	}
-
-	err = d.Set("id", interfaceMap["_key"])
-	if err != nil {
-		return err
-	}
-
-	d.SetId(b.RESTKey)
-	return nil
+	return populateEntityTypeResourceData(ctx, b, d)
```
Comment (benoittoulme) [inline provider/datasource_entity_type.go:41]: 👍

**Review by benoittoulme — APPROVED** [review]
nice and clean

File: docs/resources/entity_type.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    matching_entity_fields {
+      pod-name      = "pod-name"
+      pod-namespace = "pod-namespace"
+    }
+    metric_name = "Average Memory Usage"
+    search      = "| mstats avg(kube.pod.memory.working_set_bytes) as val WHERE `itsi_entity_type_k8s_pod_metrics_indexes` by pod-name, pod-namespace span=5m"
+    unit        = "Bytes"
+  }
+}
+```
```
Comment (dmitrykruglov) [inline docs/resources/entity_type.md:91]: Thanks for the extensive docs!

File: provider/resource_entity_type.go [hunk 3]
```diff
@@ -0,0 +1,682 @@
...
+						Optional:    true,
+						Default:     false,
+						Description: "if alert is enabled",
+					},
+					"warning_threshold": {
+						Type:     schema.TypeInt,
+						Required: true,
+					},
+					"critical_threshold": {
+						Type:     schema.TypeInt,
```
Comment (dmitrykruglov) [inline provider/resource_entity_type.go:252]: Can't this, as well as the `warning_threshold`, be a float number?

Comment (masha1997) [reply provider/resource_entity_type.go:252]: Nope, at least UI argues with  'should be Int'

Comment (masha1997) [reply provider/resource_entity_type.go:252]:
```
Invalid literal for int() with base 10: '6.5'
```

**Review by dmitrykruglov — APPROVED** [review]
Thanks, this is great!

**Pull request merged**